snap/snapcraft.yaml

```
@@ -151,6 +155,11 @@ parts:
       - jq
       - pciutils
 
+  ssh:
+    plugin: nil
+    stage-packages:
+      - openssh-client
```

To enable password authentication:
```suggestion
      - openssh-client
      - ssh-askpass
```